### PR TITLE
Add voter app code

### PR DESCRIPTION
**Pull request by lukitsbrian**
Add code for voter app

### CODE REVIEW

**Review by marbar3778 — CHANGES_REQUESTED** [review]
what is the reasoning to have starport in the name? I feel like we can easily do without it

File: scavenge/scavenge/type.diff [hunk 1]
```diff
@@ -1,18 +1,18 @@
-diff --git a/starport-scavenge/scavenge/vue/src/store/app.js b/starport-scavenge/scavenge/vue/src/store/app.js
+diff --git a/scavenge/scavenge/vue/src/store/app.js b/scavenge/scavenge/vue/src/store/app.js
```
Comment (marbar3778) [inline scavenge/scavenge/type.diff:1]: never seen this file before. do we need it?

**Review by marbar3778 — APPROVED** [review]
utACK

File: scavenge/scavenge/type.diff (hunk 1 above)
Comment (lukitsbrian) [reply scavenge/scavenge/type.diff]: Removed it!

**Pull request merged**